tests/regression/pmodel/test_pmodel.py

```diff
@@ -211,7 +211,7 @@ def test_calc_gammastar(values, tc, patm, context_manager, expvals):
     from pyrealm.pmodel import calc_gammastar
 
     with context_manager:
-        ret = calc_gammastar(tc=values[tc], patm=values[patm])
+        ret = calc_gammastar(tk=values[tc] + 273.15, patm=values[patm])
```

Comment:
Would it make sense to make 273.15 a constant, and maybe even provide a conversion function?

Reply:
Ha - it already is a constant in CoreConst. I was just being lazy in the tests.